Gems/AWSCore/cdk/app.py

```diff
@@ -37,7 +37,7 @@
 
 app = core.App()
 
-core = AWSCore(
+core_construct = AWSCore(
```

Comment:
Minor: I believe there was discussion talking about using construct or stack? seems like here the coding standard is inconsistent, core is a construct, but example is a stack.

Reply:
Seems out of scope for this change. We can address this in a separate ticket.